src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -41,23 +43,38 @@ object WorkflowActor {
 
   val AkkaTimeout = 5 seconds
 
-  type ExecutionStore = Map[Call, ExecutionStatus.Value]
+  type ExecutionStore = Map[ExecutionStoreKey, ExecutionStatus]
+  type ExecutionStoreEntry = (ExecutionStoreKey, ExecutionStatus)
+
+  // See also `WorkflowActor.createStore` that is similar, but uses the database.
+  // Currently only used for testing. May be merged / replaced with createStore.
```

Comment:
Would people be okay with removing this method and the corresponding test that uses it?  It's basically testing a path of the code that's not used.  I think this was one of the first pieces of code written for scatter support so it probably isn't relevant anymore.